src/Behavioral.Automation/Bindings/TableBinding.cs

```diff
@@ -38,10 +38,15 @@ public void CheckTableContainsRows(ITableWrapper gridRows, string behavior, Tabl
             }
             else
             {
+                if (strictCondition == "rows only")
+                {
+                    Assert.ShouldBecome(() => gridRows.CellsText.Count() == expectedValues.Count, true,
+                        $"{gridRows.Caption} is {gridRows.CellsText.Aggregate((x, y) => $"{x}, {y}")}");

```

Comment:
I would expect to see the message like this:
"Expected number of rows is ..., but actual is ..."